build/templates/_converters.py.mako

```diff
@@ -324,3 +326,50 @@ def convert_double_each_element(numbers):
 
 
 % endif
+% for enum_name in sorted(helper.filter_codegen_enums(enums)):
+    % if enums[enum_name].get('use_converter', False):
+<%
+    enum_name_in_snakecase = helper.camelcase_to_snakecase(enums[enum_name]['python_name'])
```

Comment:
I have modified the internal metadata expansion logic to generate the `enum_to_converted_value_function_name` and `converted_value_to_enum_function_name` keys instead of `use_converter`. 

Nevertheless, I have set the expansion logic to **not** generate these keys if the enum doesn't use converter instead of expanding them to None by default (the codegen logic here would treat these keys as optional) in order to avoid breaking the existing metadata for all other drivers and requiring metadata updates for all of them.

I have added unit tests for the internal metadata expansion logic but not here since this code lives in the mako template and can only be tested after the codegen is performed.